src/components/HotwaxLogo.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <figure>
+    <img class="light" src="../assets/images/hc.png"/>
+    <img class="dark" src="../assets/images/HWCLogoDarkMode.png"/>
+    </figure>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: "HotwaxLogo",
```

Comment:
Lets change this to a generic name like "Logo" so you can put any logo here.

Reply:
Done